_content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Pathways of Buddhist Thought: Four Essays"
+authors:
+  - nyanamoli
+external_url: "https://www.bps.lk/olib/wh/wh052_Nyanamoli_Pathways-Of-Buddhist-Thought--Four-Essays.html"
+drive_links:
+  - "https://drive.google.com/file/d/10nsDXmod0VTuhqKjpj7fOKUrKc_-U9-e/view?usp=sharing"
+  - "https://drive.google.com/file/d/1AVDLjlcrasmBvA01FLJjTFvt-yIxsTHh/view?usp=drive_link"
+formats: [pdf]
```

Comment:
please remove this line (the drive links are now [pdf, epub] but that's the default value for formats anyway)